graylog2-server/src/main/java/org/graylog2/shared/buffers/processors/DecodingProcessor.java

```diff
@@ -143,13 +146,7 @@ private void processMessage(final MessageEvent event) throws ExecutionException
                 message = codec.decode(raw);
             }
         } catch (RuntimeException e) {
-            String remote = "unknown source";
-            final ResolvableInetSocketAddress remoteAddress = raw.getRemoteAddress();
-            if (remoteAddress != null) {
-                remote = remoteAddress.getInetSocketAddress().toString();
-            }
-            LOG.error("Unable to decode raw message {} (journal offset {}) encoded as {} received from {}.",
-                      raw.getId(), raw.getJournalOffset(), raw.getCodecName(), remote);
+            LOG.error("Unable to decode raw message on input <{}>.", raw, inputIdOnCurrentNode);
```

Comment:
@kroepke `RawMessage#toString()` includes all relevant information except for the input ID:
```java
    public String toString() {
        final MoreObjects.ToStringHelper helper = MoreObjects.toStringHelper(this);
        helper.add("id", getId())
                .add("journalOffset", getJournalOffset())
                .add("codec", getCodecName())
                .add("payloadSize", getPayload().length)
                .add("timestamp", getTimestamp());
        if (getRemoteAddress() != null) {
            helper.add("remoteAddress", getRemoteAddress().getInetSocketAddress().toString());
        }
        return helper.toString();
    }
```